fe2-android/app/src/main/java/com/github/dedis/popstellar/model/objects/Chirp.java

```diff
@@ -6,32 +6,57 @@
 import com.github.dedis.popstellar.model.objects.security.MessageID;
 import com.github.dedis.popstellar.model.objects.security.PublicKey;
 
+import java.util.Objects;
+
 /** Class modeling a Chirp */
 public class Chirp implements Copyable<Chirp> {
 
   private static final int MAX_CHIRP_CHARS = 300;
 
-  private MessageID id;
-  private Channel channel;
+  private final MessageID id;
+  private final PublicKey sender;
+  private final String text;
+  private final long timestamp;
+  private final boolean isDeleted;
+  private final MessageID parentId;
```

Comment:
what's the parent ID ? I don't recall this in the specification.

Reply:
I don't know for sure, I only followed the already existing impmentation.

My guess is that it is the ID for the Chirp it is "replying" to.